Dockerfile

```diff
@@ -4,10 +4,11 @@ ENV NODE_ENV=production
 ENV PORT=8099
 
 VOLUME /config
-WORKDIR /app
+WORKDIR /home/node
+USER NODE
```

Comment:
the `USER node`  statement needs to be after `npm install` otherwise the installation fails.

Reply:
Won't that cause user rights issues if the `node` user do not have sufficient rights on the node_modules folder?

Reply:
Ok I changed it and a few other things following https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md